src/modules/person/api/person.controller.ts

```diff
@@ -582,4 +583,40 @@ export class PersonController {
 
         return { ok: true, value: changeUserPasswordResult.value };
     }
+
+    @Patch('uem-password')
+    @HttpCode(HttpStatus.ACCEPTED)
+    @ApiAcceptedResponse({ description: 'UEM-password for person was successfully reset.', type: String })
+    @ApiNotFoundResponse({ description: 'The person does not exist or insufficient permissions to update person.' })
+    @ApiInternalServerErrorResponse({ description: 'Internal server error.' })
+    @UseInterceptors(ResultInterceptor)
+    public async resetUEMPassword(@Permissions() permissions: PersonPermissions): Promise<Result<string>> {
+        const personId: string = permissions.personFields.id;
+        const personResult: Result<Person<true>> = await this.personRepository.getPersonIfAllowedOrRequesterIsPerson(
+            personId,
+            permissions,
+        );
```

Comment:
Is there any reason to use this repo method instead of findById? Since this endpoint can only be called for oneself, we don't need to check permissions. 

The `getPersonIfAllowedOrRequesterIsPerson` in the other endpoint can also be replaced with `getPersonIfAllowed`